app/Jobs/Emails/PresentationSubmissions/SelectionProcess/PresentationSubmitterSelectionProcessAcceptedAlternateEmail.php

```diff
@@ -0,0 +1,60 @@
+<?php namespace App\Jobs\Emails\PresentationSubmissions\SelectionProcess;
+/**
+ * Copyright 2023 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+use Illuminate\Support\Facades\Log;
+use models\main\Member;
+use models\summit\Summit;
+use utils\Filter;
+
+/**
+ * Class PresentationSubmitterSelectionProcessAcceptedAlternateEmail
+ * @package App\Jobs\Emails\PresentationSubmissions\SelectionProcess
+ */
+class PresentationSubmitterSelectionProcessAcceptedAlternateEmail extends PresentationSubmitterSelectionProcessEmail
```

Comment:
u forgot to seed all templates on
SummitEmailFlowTypeSeeder and SummitEmailFlowEventSeeder
also u need to create a migration to seed all flows on each summit on DB

Reply:
done